src/Service/EloquaApiClient.php

```diff
@@ -283,7 +283,7 @@ public function getBaseUrl() {
    *   The endpoint, when called using basic authentication or OAuth,
    *   will return details about the URLs you should be using.
    */
-  private function doBaseUrlRequest() {
+  protected function doBaseUrlRequest() {
```

Comment:
Visibility of these methods were changed so that they can be stubbed in mocked tests here - https://github.com/dakkusingh/eloqua_api_redux/pull/3/files#diff-00ee56a270baf2089b72d1d30b26d02fR250